library/Icingadb/Widget/Detail/CustomVarsDetail.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget\Detail;
+
+use Icinga\Module\Icingadb\Model\User;
+
+class CustomVarsDetail extends ObjectDetail
```

Comment:
Not a fan of the name to be honest. This is either a UserDetail or UserGroupDetail.

That it extends ObjectDetail is also not fitting. This implies that all inherited methods should work, which they don't.

Please use two different detail implementations (`UserDetail` and `UserGroupDetail`) and implement `createCustomVars()` for both and `createUserList()` for the latter.